lib/geocoder.js

```diff
@@ -61,21 +61,32 @@ Geocoder.prototype.reverse = function(query, callback) {
 * @return promise
 */
 Geocoder.prototype.batchGeocode = function(values, callback) {
-  return BPromise.resolve(values)
+  return BPromise.resolve()
```

Comment:
I think it will make sense to move that part to the abstract geocoder and to rely on the existence of `_batchGeocoder` instead of relying on throwing an error to have a fallback

Reply:
It does make sense, in fact, it was my first approach but I can't remember why I've fallen back to this way. I'll give it another try

Reply:
Done 🙂